[2025-01-29T12:18:19.236+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T12:18:19.389+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.customer_Transform manual__2025-01-29T12:17:16.105169+00:00 [queued]>
[2025-01-29T12:18:19.429+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.customer_Transform manual__2025-01-29T12:17:16.105169+00:00 [queued]>
[2025-01-29T12:18:19.431+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 3
[2025-01-29T12:18:19.780+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): customer_Transform> on 2025-01-29 12:17:16.105169+00:00
[2025-01-29T12:18:20.074+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2733) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T12:18:20.464+0000] {standard_task_runner.py:72} INFO - Started process 2771 to run task
[2025-01-29T12:18:20.345+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'customer_Transform', 'manual__2025-01-29T12:17:16.105169+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpzqwwkht3']
[2025-01-29T12:18:20.668+0000] {standard_task_runner.py:105} INFO - Job 69: Subtask customer_Transform
[2025-01-29T12:18:21.786+0000] {task_command.py:467} INFO - Running <TaskInstance: main_dag.customer_Transform manual__2025-01-29T12:17:16.105169+00:00 [running]> on host ae52aba2dca2
[2025-01-29T12:18:23.926+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='itsMe!!' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='customer_Transform' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T12:17:16.105169+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T12:17:16.105169+00:00'
[2025-01-29T12:18:24.037+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T12:18:34.051+0000] {logging_mixin.py:190} INFO - <class 'pandas.core.frame.DataFrame'>
Index: 638 entries, 0 to 2586
Data columns (total 3 columns):
 #   Column         Non-Null Count  Dtype 
---  ------         --------------  ----- 
 0   Customer ID    638 non-null    object
 1   Customer Name  638 non-null    object
 2   Segment        638 non-null    object
dtypes: object(3)
memory usage: 19.9+ KB
[2025-01-29T12:18:34.082+0000] {logging_mixin.py:190} INFO - None
[2025-01-29T12:18:35.391+0000] {logging_mixin.py:190} INFO - !_upload_to_S3_complete_!
[2025-01-29T12:18:35.440+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-29T12:18:35.999+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T12:18:36.042+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=customer_Transform, run_id=manual__2025-01-29T12:17:16.105169+00:00, execution_date=20250129T121716, start_date=20250129T121819, end_date=20250129T121836
[2025-01-29T12:18:36.647+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T12:18:37.286+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-29T12:18:37.316+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
